Lib/mimetypes.py

```diff
@@ -32,6 +32,8 @@
 except ImportError:
     _winreg = None
 
+from collections import OrderedDict
```

Comment:
This no longer needed in 3.7+.

Reply:
Should I remove all usage of OrderedDict from the PR and let others add it back port it to 2.7? Or should I do something like set OrderedDict = dict if Python version > 3.7?

Reply:
The order of the items in the list is a critical part of the fix - I'd vote for retaining the explicit use of OrderedDict.

Reply:
Adding OrderedDict makes this PR more complex, and the resulting code clumsy, and this is absolutely unneeded in 3.7+. I think it would be better to merge a simple fix into master and 3.7, and prepare more complex backport for 3.6.

Reply:
Since 3.6 is only receiving security fixes, there's no need to worry about the backport to 3.6 anymore.  So, you can remove the OrderedDict as this will only be in 3.7 and 3.8.